src/model/Project.ts

```diff
@@ -307,7 +307,7 @@ export default class Project extends Ressource {
    *
    * @return Activity[]
    */
-  getActivities(types: Array<string>, starts_at: number) {
+  getActivities(types: Array<string>, starts_at: string|boolean) {
```

Comment:
@pagum while we are here already, can we make start_at?: Date and then we can convert the Date object into date string here